.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+
+node_js: 
+  - "lts/*"
+
+matrix:
+  include:
+    - os: linux
+    - os: osx
+
+before_install:
+  # Set up X Virtual Framebuffer because an instance of VS Code runs the tests
+  # See https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
```

Comment:
1. What's the difference between this and `xvfb-run npm test --silent` from our `buildspec.yaml`?
2. Why isn't this needed for osx?

Reply:
Also, why isn't this needed for osx?